custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc_daily.py

```diff
@@ -72,11 +78,6 @@ def aggregation_query(self):
             ('cases_person_adolescent_girls_15_18_all',),
             ('daily_attendance_open', '0'),
             ('num_awcs',),
-            ('num_launched_states',),
```

Comment:
For this for columns. Can you try this.
Instead of removing these  do them like this `('num_launched_states','0'),`. I think if you do so. the test cases failures will go away and the changes you made on the `agg_awc_daily_2017-05-28_sorted.csv` will not be required.

Reply:
Alright, I try this change, test and then update the PR accordingly.

Reply:
After making suggested changes and testing, updating this PR with updated code. Please have a look. Thanks!